src/main/java/Waiter.java

```diff
@@ -0,0 +1,8 @@
+import java.util.concurrent.TimeUnit;
+
+public class Waiter {
+
+    public static void explicitWait(int waitTime) {
```

Comment:
It is not explicit wait. Explicit wait refers to visibility/text appearance/etc. Please update this method to be real explicit waiter and then use it where needed

Reply:
renaming to implicitWait as I am not waiting for any element on the Page. Mail.ru is just a crazy page for loading and don't have enough time to track which elements I should wait. So need just configurable method for implicit wait.

Reply:
Okay, just update implicit waiter back to default value after that specific place, please.